llarp/router/router.cpp

```diff
@@ -150,19 +150,13 @@ namespace llarp
       return _rcLookupHandler.GetRandomWhitelistRouter(router);
     }
 
-    auto pick_router = [&](auto& collection) -> bool {
-      const auto sz = collection.size();
-      auto itr = collection.begin();
-      if (sz == 0)
-        return false;
-      if (sz > 1)
-        std::advance(itr, randint() % sz);
-      router = itr->first;
+    if (const auto maybe = nodedb()->GetIf([](const auto&) -> bool { return true; }, true);
```

Comment:
If this usage of `GetIf` is correct, perhaps it should be renamed?  Perhaps `GetRandomIf` or similar?